app/views/social_networking/homes/show.html.erb

```diff
@@ -26,11 +26,15 @@
     <li class="list-group-item">
       <button class="btn btn-link" ng-click="home.newOnYourMindStatement()">What's on your mind?</button>
     </li>
-    <li class="list-group-item" ng-repeat="item in home.actionItems">
+    <li class="list-group-item" ng-repeat="item in home.actionItems" ng-if="home.actionItems.length > 0">
```

Comment:
Right, but I'm asking because it seems like ` ng-if="home.actionItems.length > 0"` is redundant here.